File: Components/Reporting/VB/DevExpress.DemoReports/OddEvenStyles/Report.vb
Imports System
Imports System.ComponentModel
Imports System.Collections
Imports System.Diagnostics
Imports DevExpress.XtraReports.UI
Imports System.IO

Namespace XtraReportsDemos.OddEvenStyles
    Partial Public Class Report
        Inherits DevExpress.XtraReports.UI.XtraReport

        Public Sub New()
            InitializeComponent()

            Name = ReportNames.OddEvenStyles
            DisplayName = ReportNames.OddEvenStyles
            ' this.productsTableAdapter1.Fill(this.dsOddEvenStyles1.Products);
        End Sub

    End Class
End Namespace

File: Components/Reporting/CS/DevExpress.DemoReports/PivotGrid/Report.cs
using System;

namespace XtraReportsDemos.PivotGrid
{
    public partial class Report : DevExpress.XtraReports.UI.XtraReport {

		public Report() {
			InitializeComponent();
            Name = ReportNames.PivotGrid;
            DisplayName = ReportNames.PivotGrid;
		}
	}
}
